<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Manuel A. del Campo</author>
 <class>Panel</class>
 <widget class="QWidget" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>38</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>16</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>101010</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>38</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Panel{
#background-color:#333;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="StartButton" name="startButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#startButton{
	width: 70;
    background-color: #0784c5;
	border: 1px solid #0784c5;
    padding: 0;
    padding: 5px 12px 7px 15px;
}
QToolButton#startButton:hover{
    background-color: #07689a;
        border: 1px solid #07689a;
}
QToolButton#startButton:pressed{
   background-color: #07689a;
	border: 1px solid #07689a;
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="quickLaunchArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>120</horstretch>
       <verstretch>38</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#quickBar{
        border-bottom: 1px solid;
        background-color:#333;
        border-bottom-color:#272727;
        /*border-right: 1px solid #474747;*/
        padding: 3px;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="windowsArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>38</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border-bottom: 1px solid;
background-color:#333;
border-bottom-color:#272727;
/*border-left: 1px solid #272727;*/
padding: 1px ;
}

QToolButton{
	image-position: left;
	background-color: #1a1a1a;
	font-size: 12pt;
    padding: 0;
    color: palette(dark);
    padding: 4;
    list-style-type: none;
	border-top: 2px solid #AAA;
    line-height: 24px;
	text-align: right;
	
}
QToolButton:hover{
   border-top: 2px solid #0784c5;
}
QToolButton:actived{
    	border-top: 2px solid #0784c5;
	background-color: #1a1a1a;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="systrayArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>38</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget #systrayArea{
	background-color:#333;
	    padding-top: 2px;
	    padding-bottom: 2px;
}

QFrame{
	width: 32px;
	height: 32px;
    
	background-color: #3c3c3c;

    color: #AAA;
        list-style-type: none;
        border: 1px solid #3c3c3c;
    line-height: 24px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="clockArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
/*border-bottom: 1px solid;*/
background-color:#333;
border-bottom-color:#272727;
/*border-left: 1px solid #272727;*/
}
QLabel{
        color:white;
        border: 0px solid;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StartButton</class>
   <extends>QToolButton</extends>
   <header>c++/StartButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
